Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimTokensViewController.swift

```diff
@@ -203,7 +203,8 @@ class ClaimTokensViewController: LoadableViewController {
     override func reloadData() {
         super.reloadData()
 
-        timestamp = Date().timeIntervalSince1970
+        // This to make sure that the last block time interval is less than the used one
+        timestamp = Date().timeIntervalSince1970 - 30
```

Comment:
I would extract it to a func with a clear name. Like lastBlockTimestampEstimation()